illumio.json

```diff
@@ -11,7 +11,7 @@
     "product_version_regex": ".*",
     "publisher": "Illumio",
     "license": "Copyright (c) Illumio, 2022",
-    "app_version": "1.0.0",
+    "app_version": "1.0.1",
```

Comment:
What version are we expecting to go with for release?

Reply:
Changed